src/slic3r/GUI/GLCanvas3D.cpp

```diff
@@ -6419,7 +6419,7 @@ void GLCanvas3D::_resize(unsigned int w, unsigned int h)
     m_last_w = w;
     m_last_h = h;
 
-    const float font_size = 1.5f * wxGetApp().em_unit();
```

Comment:
Will it affect normal or other scalling?

Reply:
It "should" only affect text and (so far) I haven't found any other items affected by the change in scaling.  BUT...  :-)

I've tried to excersize as much of the imgui bits in the UI as I can using different Windows 11 scaling values.  But I do not have Mac or Linux boxes set up currently, so I have not been able to check there.  I can get a Linux box up fairly quickly if you'd like me to check there as well.  A Mac would be a bit more difficult, but doable.

And, more importantly, I'm still waiting for my first 3d printer to be delivered!  So, I haven't even been able to actually print anything.  I've just been trying to get familar with Bambu Studio and Orca Slicer so I'll be able to print my first Benchy. :-)

I'll be sticking to Orca Slicer though since it has the resizable panes which I need with this darn 350% display scaling I have to use.

Reply:
@lllucius
Hehehe, enjoy 3d printing.

I'm not clear about the issue and the cause as well as the proposed changes here.
Is the issue caused by the font becomes too large when display scaling > 300%?
If so, cap the font_size to e.g 32 is better.

Reply:
@lllucius
Have fun!

Would it be better if we simply cap the font size here?
Just so that to reduce the overlapping zone.
e.g. 
#ifdef _WIN32
    font_size  = std::max(font_size*1.5f, 32);
...
#endif

Reply:
Unfortunately, limiting the font size would somewhat defeat the purpose of having a higher display scaling factor as the text could be too small at higher resolutions.

There's actually one other issue that I've found (finally go my printer) when using 350% scaling.  You are unable to send a print to the device since the "Send" button falls off the bottom of the screen when using 3840x2160 and 350% scaling.

Reply:
> There's actually one other issue that I've found (finally go my printer) when using 350% scaling. You are unable to send a print to the device since the "Send" button falls off the bottom of the screen when using 3840x2160 and 350% scaling.

Yeah, it's really rare to have such high DPI scalling.
Are you running on a Steam deck or similar device?

Reply:
I know it seems crazy, but I actually have a 32" monitor.  Just have a couple of crappy eyeballs to go along with that.  :-)  

I do know of one other person who wants to use Orca that's in the same (similar) situation.

Reply:
Oh, and thanks much for merging!